script.js

```diff
@@ -0,0 +1,86 @@
+const ham = document.getElementById('ham-open');
+const cancel = document.getElementById('cancel-ham');
+const nav = document.getElementById('link-and-nav');
+const speaker = document.getElementById('speak');
+let check = true;
+const li = document.createElement('li');
+li.setAttribute('class', 'flex');
+const ul = document.createElement('ul');
+
+// function Close and OPen Ham munu
+function closeOpenHam() {
+  nav.classList.toggle('hide-mobile');
+}
+
+// Event Lisner for Ham and Close buttons
+ham.addEventListener('click', closeOpenHam);
+cancel.addEventListener('click', closeOpenHam);
+
+// Object to store speakers
+const speakersObj = {
+  speakerOne: {
+    name: 'John Deo',
+    accomplishment: 'Well known Fashion tycoon, with blinding dress style',
+    studied: 'Deo studed fashion in bakely college of fashion freaks',
+    picture: './images/guestOne.jpg',
+  },
+  speackerTwo: {
+    name: 'Albert Ishmil',
+    accomplishment: 'Blazing Hot and always ready to slay aka (The ladies Man)',
+    studied: 'Ishmil studied fashion in hervard university south carolina',
+    picture: './images/guestTwo.jpg',
+  },
+  speackerThree: {
```

Comment:
You have done a great job making these dynamic and rendering them well 👍🏻💯

Reply:
Am i to do any changes here ?

Reply:
@Ginohmk we are just acknowledging your great effort here:+1: No change to make on those lines.